recipes/pysurvival/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pysurvival" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 57cb567323fa314ceaf17fe39745b007855606cb1304a4124a78334ff9d03707
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py>38 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - matplotlib-base
+    - pandas
+    - pip
```

Comment:
It is very likely that pip is not a runtime dependency.

Reply:
I have removed it in the new commit, but I think the extensions are an issue.